spec/models/follow_spec.rb

```diff
@@ -57,7 +57,7 @@
     it "creates a chat channel when users follow mutually" do
       described_class.create!(follower: user_2, followable: user)
       expect do
-        perform_enqueued_jobs do
+        Sidekiq::Testing.inline! do
```

Comment:
We actually have a helper for this `sidekiq_perform_enqueued_jobs` which you could use here.

Reply:
sure thing, I will modify this.